ReactTask/src/components/BtnsComponents/BtnSubmitFormSignIn.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styled from 'styled-components';
+import { StylBtnForm } from './../common/Btns/';
+import { StylIconArrowBtn } from './../common/icons/';
+
+const BtnForm = styled(StylBtnForm)`
+  margin-bottom: 32px;
+`;
+
+const BtnSubmitForm = ({ text }) => {
+  return (
+    <BtnForm>
+      {text}
+      <StylIconArrowBtn />
+    </BtnForm>
+  );
+};
+
+export default BtnSubmitForm;
```

Comment:
Code duplicate with BtnSubmitForm.js, probably have to create one button for all forms

Reply:
The `margin-bottom` style is added to the `BtnFormSignIn` button

Reply:
Added basic style `margin-bottom: 30px`